### PR TITLE
Mirror monitor and template updates (+ more)

### PR DESCRIPTION
**Pull request by jonathanspw**
- Change to aioredis lib w/hiredis
- Update Sentry SDK
- Enable Sentry performance tracking
- Enable Sentry aiohttp integration
- Disable Sentry redis integration (doesn't support aioredis yet)
- Update mirror monitoring interval to 1h
- Track mirror flapping.
  - A mirror is marked as flapping when it has failed a health check within the past 12 hours.  Mirrors marked as flapping are still displayed on the mirrors page but are not served as part of `/mirrorlist` requests.
- Improve DB call efficiency for listing all mirrors.  The result is the page generation time is now under 50ms on average instead of 350ms+
- Improve DB call efficiency for mirrorlist requests.  The result is the generation time is on average 50-100ms faster than before.
- Add tracking of which mirrors support IPv6
- Add generic favicon to prevent browsers from hitting 404s when checking default URLs for `/favicon.ico`

### CODE REVIEW

**Comment by jonathanspw** [comment]
image

File: src/backend/db/models.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ class Mirror(Base):
     ip = Column(String, nullable=False)
     latitude = Column(Float, nullable=False)
     longitude = Column(Float, nullable=False)
-    is_expired = Column(Boolean, nullable=False, default=False)
+    status = Column(String, nullable=False)
```
Comment (soksanichenko) [inline src/backend/db/models.py:131]: Did you make db migrations?

File: src/backend/templates/mirrors.html [hunk 2]
```diff
@@ -94,6 +94,11 @@
                             {% endif %}
                         </td>
                     {% endfor %}
+                    <td>
+                        {% if mirror.ipv6 %}
```
Comment (soksanichenko) [inline src/backend/templates/mirrors.html:98]: Did you add the column to header?

**Review by soksanichenko — CHANGES_REQUESTED** [review]

File: src/backend/db/models.py (hunk 1 above)
Comment (jonathanspw) [reply src/backend/db/models.py:131]: sqlite cannot change column type so no I didn't.  Do we want one that simply removes the is_expired col and adds status?

File: src/backend/templates/mirrors.html (hunk 2 above)
Comment (jonathanspw) [reply src/backend/templates/mirrors.html:98]: Yep https://github.com/AlmaLinux/mirrors/pull/291/commits/983abaf6198191a924e5546e6e6b29393c3df0b8#diff-7dc0ea0263421b070b14c289b52f9467996fb61796dbd1277b1c99ea1da14e4fR143

File: src/backend/db/models.py (hunk 1 above)
Comment (soksanichenko) [reply src/backend/db/models.py:131]: I did migration to postgres which can do it)

Comment (soksanichenko) [reply src/backend/db/models.py:131]: Never mind. I just mix up services.

**Review by soksanichenko — APPROVED** [review]

**Pull request merged**